File: src/library/scala/runtime/java8/JFunction0$mcF$sp.java

/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */

package scala.runtime.java8;

@FunctionalInterface
public interface JFunction0$mcF$sp extends scala.Function0, java.io.Serializable {
    float apply$mcF$sp();

    default Object apply() { return scala.runtime.BoxesRunTime.boxToFloat(apply$mcF$sp()); }
}

File: src/library/scala/runtime/java8/JFunction0$mcF$sp.scala
/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */
package scala.runtime.java8

@FunctionalInterface trait JFunction0$mcF$sp extends Function0[Any] with Serializable {
  def apply$mcF$sp: Float
  override def apply: Any = scala.runtime.BoxesRunTime.boxToFloat(apply$mcF$sp)
}